websites/T/TOD/presence.ts

```diff
@@ -0,0 +1,281 @@
+class TOD extends Presence {
+	constructor(presenceOptions: PresenceOptions) {
+		super(presenceOptions);
+	}
+
+	getVideo() {
+		return document.querySelector<HTMLMediaElement>(
+			"video"
+		);
+	}
+
+	getVideoType() {
+		if (this.getVideoTitle().match(/S\d+\s*\|\s*E\d+/i) || this.getVideoTitle().match(/EP\s*\d+/i))
+			return "show";
+		 else 
+			return "movie";
+	}
+
+	getTitle(eyebrow = false) {
+		if (this.isWatching() && eyebrow) return this.getVideoTitle();
+
+		const title = document.querySelector("div.bein-dh1-hero__head-container > h1.bein-dh1-hero__title.heading-shadow")?.textContent;
+
+		return (
+			title ??
+			this.getVideoTitle().replace(/\d+\s*-\s*\d+/ as RegExp, "vs") ??
+			"Unknown"
+		);
+	}
+
+	getVideoTitle() {
+		return document.querySelector(
+			"div.diva-standard-title"
+		)?.textContent;
+	}
+
+	isWatching() {
+		return !!this.getVideoTitle();
+	}
+
+	isLive() {
+		if (document.querySelector("span.diva-live-now"))
+			return true;
+		else
+			return false;
+	}
+
+	isTrailer() {
+		return this.getVideoTitle().includes("Trailer") ? true : false;
```

Comment:
`.includes` already returns a boolean
```suggestion
		return this.getVideoTitle().includes("Trailer");
```